packages/components/src/internal/components/picklist/PicklistEditModal.tsx

```diff
@@ -26,10 +26,21 @@ interface Props {
     onCancel: () => void;
     onFinish: (picklist: Picklist) => void;
     showNotification?: boolean;
+    getPicklistURL?: (picklistId: number) => string;
```

Comment:
There is a lot of this passing around of the `getPicklistURL` helper. I see in the inventory PR the implementation for the helper function. Is the idea to be able to build the right URL from either the SM app or the FM app so that the user is sent back to the picklist in the SM app? 

If so, would it be possible to instead do something along the lines of `getCurrentProductName` in which we infer which app we are in and then create the URL accordingly from within these shared components so that it isn't up to the callers to tell us that information?